Updates GCP rights for buckets and SPN (#221)
meshcloud/meshcloud-docs: docs/meshstack.gcp.landing-zones.md

> The maximum filesize currently is 1MB please [contact us](mailto:support@meshcloud.io) if you need support for bigger template configurations.

> The replicator needs to assign the project service accounts read access to the bucket so the templates can be fetched. Its therefore necessairy to give the meshfed service account the following permissions on the storage bucket:

Reviewer: ```suggestion
The replicator needs to assign the project service accounts read access to the bucket so the templates can be fetched. It is therefore necessary to give the meshfed service account the following permissions on the storage bucket:
```

```text
storage.buckets.setIamPolicy
storage.buckets.getIamPolicy
storage.objects.get
storage.objects.list
storage.buckets.list
storage.buckets.get
```

> It might be a good idea to create a custom role containing this conditions. The replicator then assignes read access for the projects service accounts which have the form of `<PROJECT_ID>@cloudservices.gserviceaccount.com`.

Reviewer 2: ```suggestion
We strongly suggest to create a custom role containing this conditions. The replicator then assignes read access for the projects service accounts which have the form of `<PROJECT_ID>@cloudservices.gserviceaccount.com`.
```
```suggestion
It might be a good idea to create a custom role containing this conditions. The replicator then assignes read access for the projects service accounts which have the form of `<PROJECT_ID>@cloudservices.gserviceaccount.com`.
```

Reviewer: Maybe "We recommend". At the end of the day there is no huge difference between the two otherwise :D

Author: yeah second that. In the end it dosnt really matter... (pun intended) and its up to the customer to decide if he wants to use multiple built in roles or a custom role.
